src/collapse/collapse.ts

```diff
@@ -0,0 +1,28 @@
+import {Directive, Input} from 'angular2/angular2';
+
+@Directive({
+  selector: '[ngb-collapse]',
+  host: {'class': 'collapse', '[class.in]': '_open', '[attr.aria-expanded]': '_open', '[style.height.px]': '_height'}
+})
+export class NgbCollapse {
+  private _open: boolean;
+  private _height: number;
```

Comment:
The problem is that in angular 2 you can do stuff like:

```
<div [ngb-collapse]="foo" #collapse>
...
</div>
{{ collapse.height }}
```

That being said and after I explained this to you, perhaps `height` makes more sense as private. I will leave that to discussion of my fellow team members. For now, leave it as is.